wfe2/wfe_test.go

```diff
@@ -3530,6 +3564,13 @@ func TestARI(t *testing.T) {
 	wfe.RenewalInfo(context.Background(), event, resp, req)
 	test.AssertEquals(t, resp.Code, 200)
 	test.AssertEquals(t, resp.Header().Get("Retry-After"), "21600")
+	var ri core.RenewalInfo
+	err = json.Unmarshal(resp.Body.Bytes(), &ri)
+	test.AssertNotError(t, err, "unmarshalling renewal info")
+	// The start of the window should be in the past.
+	test.AssertEquals(t, ri.SuggestedWindow.Start.Before(wfe.clk.Now()), true)
+	// The end of the window should be in the future.
+	test.AssertEquals(t, ri.SuggestedWindow.End.After(wfe.clk.Now()), true)
```

Comment:
Is this a coincidence? Isn't the validity period of `//test/hierarchy/ee-r3.cert` wholly unrelated to the fake clock the unit tests put on `wfe.clk`?

Reply:
You're absolutely right, I'll go ahead and remove these test cases.

Reply:
Replaced with a simple case that ensure the window is actually a window.

Reply:
Sounds good. You could also check that they're both after the notBefore date and before the notAfter date.

Reply:
Oh, yeah that makes sense and it will also continue to work even if we make changes for Flexible ARI.